src/sentry.rs

```diff
@@ -37,3 +37,7 @@ pub trait SentryFutureExt: Sized {
 }
 
 impl<F> SentryFutureExt for F {}
+
+pub trait WriteSentryScope {
```

Comment:
_nit_: Docs for this, please. I guess the trait is nice, but just a plain function would've sufficed :D